sql/core/src/test/resources/sql-tests/results/pgSQL/float8.sql.out

```diff
@@ -528,7 +528,7 @@ SELECT atanh(double('0.5'))
 -- !query 56 schema
 struct<ATANH(CAST(0.5 AS DOUBLE)):double>
 -- !query 56 output
-0.5493061443340549
+0.5493061443340548
```

Comment:
Other test values will need to change.
Takeshi says this isn't the value from PostgreSQL anyway

Reply:
Yeah, thanks! Looks nice.

Reply:
Yes. PostgreSQL  returns `0.549306144334055`:
https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/float8.out#L490-L494